_posts/2020-06-10-#19105.md

```diff
@@ -3,21 +3,68 @@ layout: pr
 date: 2020-06-10
 title: "Add Muhash3072 implementation in Python"
 pr: 19105
-authors: [fjahr]
+authors: ["sipa", "fjahr"]
 components: ["tests", "crypto"]
 host: fjahr
 status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Add notes and questions
 ## Notes
 
+- This is the second in a series of PRs implementing the Coin Statistics
+  Index. [#18000](https://github.com/bitcoin/bitcoin/pull/18000) provides
+  an overview of the work and is used to keep track of the different PRs.
+
+- The first PR in the series was [#19055](https://github.com/bitcoin/bitcoin/pull/19055)
+  and it has been featured in the PR review club [two weeks ago](https://bitcoincore.reviews/19055.html).
+  If you haven’t seen it, it would be very beneficial for you to read that
+  conversation first. There are also some links to papers in the description
+  that might help you understand the theoretical background of the algorithm
+  better.
+
+- Since then the scope of it has been reduced based on the feedback in the
+  session. Today’s PR now only adds the Python implementation of `MuHash3072`
+  which was originally in [#19055](https://github.com/bitcoin/bitcoin/pull/19055). 
+  [#19055](https://github.com/bitcoin/bitcoin/pull/19055) now only adds the C++
+  implementation.
+
+- The previous session was more heavily focussed on conceptual understanding
+  of the project as a whole. This one should be really dedicated towards
+  understanding the implementation of `MuHash3072`, both in Python and in C++.
+
+- The next PR in the series is [#19145](https://github.com/bitcoin/bitcoin/pull/19145).
+  It may be interesting to skip ahead to see how some of the code is used but
+  if you participated in the previous review club session you have already seen
```

Comment:
perhaps "if you reviewed the PR of the previous review club session then you may be familiar with it."